src/main/java/com/exasol/releaserobot/usecases/validate/GitRepositoryValidator.java

```diff
@@ -0,0 +1,196 @@
+package com.exasol.releaserobot.usecases.validate;
+
+import static com.exasol.releaserobot.usecases.ReleaseRobotConstants.VERSION_REGEX;
+
+import java.time.LocalDate;
+import java.util.*;
+import java.util.logging.Logger;
+
+import com.exasol.releaserobot.repository.*;
+import com.exasol.releaserobot.usecases.*;
+import com.exasol.releaserobot.usecases.ReportImpl.ReportName;
+
+/**
+ * Contains validations for a Git project.
+ */
+public class GitRepositoryValidator {
```

Comment:
Would be good to eventually remove all "Git" references from the `usecases` package. Also, for another PR